docs/graph.md

```diff
@@ -67,6 +67,30 @@ Edges can be added between the nodes as follows:
 </edge>
 ```
 
+## Creating graphs
+
+Graphs can be created via the graph menu:
+
+```bash
+# show graph commands
+warcli graph --help
+
+# list some built-in graph generators
+warcli graph list
+
+# lookup a graph type
+warcli graph lookup navigable_small_world_graph
+
+# Create a graph of version 25.1 nodes using graph-type-specific keyword options
+warcli graph create --version=25.1 navigable_small_world_graph n=25 p=1 q=3 r=2 dim=2
+
+# Create a random internet AS-style graph with random node versions
+warcli graph create --random random_internet_as_graph n=50
```

Comment:
should also document `outfile` option since generating a graph is kinda useless without it. or a flag to generate a gaph then network start it immedeatly

Reply:
Yeah, writing this pull made me remember we had meant to add an RPC (or inner function) which takes a graph _object_ instead of a file. Then we could pass around graph objects from e.g. a front end.